Testscripts/Windows/VALIDATE-INTEL-SGX-DRIVER.ps1

```diff
@@ -20,6 +20,17 @@ function Main {
 
     $ubuntuVersion = Run-LinuxCmd -Command "cat /etc/issue" `
         -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort
+
```

Comment:
Why don't we use this?
$global:detectedDistro -eq "UBUNTU"

Reply:
> Why don't we use this?
> $global:detectedDistro -eq "UBUNTU"

there are other version like Ubuntu 14.04 which not support, and also the initial design is from Francis, both of us did the review at that time.

Reply:
acked